internals/secrethub/rm.go

```diff
@@ -36,14 +36,14 @@ func NewRmCommand(io ui.IO, newClient newClientFunc) *RmCommand {
 }
 
 // Register registers the command, arguments and flags on the provided Registerer.
-func (cmd *RmCommand) Register(r command.Registerer) {
+func (cmd *RmCommand) Register(r cli.Registerer) {
 	clause := r.Command("rm", "Remove a directory, secret or version.")
 	clause.Alias("remove")
-	clause.Arg("path", "The path to the resource to remove (<namespace>/<repo>[/<path>])").Required().SetValue(&cmd.path)
-	clause.Flag("recursive", "Remove directories and their contents recursively.").Short('r').BoolVar(&cmd.recursive)
-	registerForceFlag(clause).BoolVar(&cmd.force)
+	clause.Flags().BoolVarP(&cmd.recursive, "recursive", "r", false, "Remove directories and their contents recursively.")
+	registerForceFlag(clause, &cmd.force)
 
-	command.BindAction(clause, cmd.Run)
+	clause.BindAction(cmd.Run)
+	clause.BindArguments([]cli.Argument{{Value: &cmd.path, Name: "path", Required: true, Placeholder: "<namespace>/<repo>[/<path>]", Description: "The path to the resource to remove."}})
```

Comment:
Similar placeholders are all defined in constants. Is there a specific reason this one isn't?

Reply:
The other placeholders work for wither a repo, a directory, a secret or a version. We have no placeholders that suggest that we can put either a directory or a secret.
Fixed by making it a constant placeholder in `path_placeholder.go`